doc/source/getting-started-linux-tutorial.rst

```diff
@@ -14,33 +14,35 @@ Volatility3 does not provide the ability to acquire memory.  Below are some exam
 Be aware that LiME raw format is not supported by volatility3, the padded or lime option should be used instead. `This issue contains further information <https://github.com/504ensicsLabs/LiME/issues/111>`_.
 
 Procedure to create symbol tables for linux
---------------------------------------------
+-------------------------------------------
 
-To create a symbol table please refer to :ref:`symbol-tables:Mac or Linux symbol tables`.
-
-.. tip:: It may be possible to locate pre-made ISF files from the `Linux ISF Server <https://isf-server.techanarchy.net/>`_ ,
-    which is built and maintained by `kevthehermit <https://twitter.com/kevthehermit>`_.
-    After creating the file or downloading it from the ISF server, place the file under the directory ``volatility3/symbols/linux``.
-    If necessary create a linux directory under the symbols directory (this will become unnecessary in future versions).
+To create a symbol table please refer to :ref:`symbol-tables:Mac or Linux symbol tables`.  
+After creating the file, place it under the directory ``volatility3/symbols``.  
+Volatility3 will automatically detect and use symbol tables from this location.
 
 
 Listing plugins
 ---------------
 
-The following is a sample of the linux plugins available for volatility3, it is not complete and more plugins may
-be added.  For a complete reference, please see the volatility 3 :doc:`list of plugins <volatility3.plugins>`.
-For plugin requests, please create an issue with a description of the requested plugin.
+Volatility3 currently supports over 40 Linux-specific plugins covering a wide range of forensic analysis needs, such as process enumeration, memory-mapped file inspection, loaded modules, and kernel tracing features.
+
+Some representative plugins include:
+
+- ``linux.pslist``: Lists running processes with their PIDs and PPIDs.
+- ``linux.bash``: Recovers bash command history from memory.
+- ``linux.lsmod``: Displays loaded kernel modules.
+- ``linux.kmsg``: Reads messages from the kernel log buffer.
+- ``linux.elfs``: Lists all memory-mapped ELF files.
+- ``linux.check_creds``: Checks for suspicious credential structures.
+- ``linux.vmayarascan``: Scans process memory using YARA signatures.
+
+For a full list of supported plugins, run the following command:
 
 .. code-block:: shell-session
 
-    $ python3 vol.py --help | grep -i linux. | head -n 5
-        banners.Banners     Attempts to identify potential linux banners in an
-        linux.bash.Bash     Recovers bash command history from memory.
-        linux.malware.check_afinfo.Check_afinfo
-        linux.malware.check_creds.Check_creds
-        linux.malware.check_idt.Check_idt
+    $ python3 vol.py --help | grep -i linux.
```

Comment:
Hi @ikelos, @eve-mem 
From this PR (#1764), We updated new version about how to run a program (`vol -f`)
However, the documents are showing old method. (`python3 vol.py -h`)
It seems that unification is needed to avoid confusion among users.
What do you think about this?

Reply:
I think in general I'd prefer to keep telling people to run it via `python vol.py` but I'm open to different views if there's a strong feeling otherwise.  My worry is people clone the repo, type vol and nothing happens.